From ee477ec067310a6a8973ca3cb273a17ddf03b2da Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Fri, 3 Nov 2017 15:12:45 -0700
Subject: [PATCH] MANTA-3205 occasional errors about picker having no
 connections Reviewed by: Tim Kordas <tim.kordas@joyent.com> Approved by:
 Jordan Hendricks <jordan.hendricks@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 16192a4..1446d2c 100644
--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
         "mahi": "2.0.1",
         "marlin": "git+https://github.com/joyent/manta-marlin.git#master",
         "mime": "1.2.11",
-        "moray": "3.3.0",
+        "moray": "3.4.1",
         "once": "1.3.0",
         "restify": "2.6.3",
         "sshpk": "1.13.1",
-- 
2.21.0

